gourmet/backends/db.py

```diff
@@ -1868,7 +1868,7 @@ def parse_ingredient (self, s, conv=None, get_key=True):
         s = s.decode('utf8').strip(
             '\u2022\u2023\u2043\u204C\u204D\u2219\u25C9\u25D8\u25E6\u2619\u2765\u2767\u29BE\u29BF\n\t #*+-')
         s = str(s)
```

Comment:
as you touch it, we may want to remove this line. The one before already generates a string and we're really not doing anything here, right?!